docs/src/pages/guide/how-to/03-building-workflows.mdx

```diff
@@ -10,7 +10,7 @@ Workflows are defined in a dedicated directory. The directory path is configured
 
 Here's an simple example of a workflow that uses predefined tools to crawl a website, uses an LLM to structure the data, and then analyzes the sentiment of the content.
 
-```ts filename="src/workflows/index.ts" showLineNumbers copy
+```ts filename="src/mastra/blueprints/index.ts" showLineNumbers copy
```

Comment:
we should be using /src/mastra/workflows not bleuprints

Reply:
Got it!